solr/core/src/test/org/apache/solr/security/BasicAuthIntegrationTest.java

```diff
@@ -114,7 +114,7 @@ public void testBasicAuth() throws Exception {
       String baseUrl = randomJetty.getBaseUrl().toString();
       verifySecurityStatus(cl, baseUrl + authcPrefix, "/errorMessages", null, 20);
       zkClient().setData("/security.json", STD_CONF.replaceAll("'", "\"").getBytes(UTF_8), true);
-      verifySecurityStatus(cl, baseUrl + authcPrefix, "authentication/class", "solr.BasicAuthPlugin", 20);
+      verifySecurityStatus(cl, baseUrl + authcPrefix, "authentication/class", "solr.BasicAuthPlugin", 20, "solr", "SolrRocks");
```

Comment:
A general comment for this test class: Why not instead add blockUnknown=false to `STD_CONF`? We do not necessarily need to test that the default is true in this test. With the current changes, the nature of the test has changed and does no longer test that certain actions can be done without credentials while other need those.

As long as we have proper testing of the behavior of the new default we should not need to change all other tests?

Reply:
The `protected static final String STD_CONF` in that test class actually *does* define an initial user...